Practice3/Decision1.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Practice3
+{
+    class Decision1
+    {
+        List<int> _list = new List<int>();
+
+        public int MinRange { get; set; }
+        public int MaxRange { get; set; }
+
+        public void SetRange()
+        {
+            Console.WriteLine("Plase set min and max value that can be added into the array.");
+            Console.WriteLine("Min value:");
+            MinRange = Convert.ToInt32(Console.ReadLine());
+
+            Console.WriteLine("Max value:");
+            MaxRange = Convert.ToInt32(Console.ReadLine());
+        }
+
+        public void AddValues()
+        {
+            while (true)
```

Comment:
please, find any way to limit this loop. we don't use infinite loops in programs/tests because it's a  week spot for a memory leaks

Reply:
I could add a counter and check that number of added values is less or equals to the countered value. But the main idea of this class was the possibility to add as many values as the user wants

Reply:
yep, please, add a counter and set any big number so the user is not limited and the program is safe 😄